src/main/java/org/commcare/suite/model/GetLimitedInstances.java

```diff
@@ -0,0 +1,42 @@
+package org.commcare.suite.model;
+
+import org.javarosa.core.model.instance.DataInstance;
+import org.javarosa.core.model.instance.ExternalDataInstance;
+
+import java.util.Enumeration;
+import java.util.Hashtable;
+import java.util.Set;
+
+public class GetLimitedInstances {
```

Comment:
Actially we can just put the method in this class into `InstanceUtils` instead of creating a new class.